src/content/Wedstrijden.json

```diff
@@ -1,5 +1,52 @@
+            "path": "/nskmeerkamp"
+        },
+        {
+            "naam": "Biermijl",
+            "datum": "oktober 2024",
+            "tijd": "volgt",
+            "locatie": "D.S.A.V. Dodeka",
+            "adres": "Sportring 12",
+            "postcode": "2616LK Delft",
+            "maps": "https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2454.76308928639!2d4.365573151907881!3d52.029413480121534!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47c5b71b1ed00f3d%3A0xb952856aa5bdefd3!2sD.S.A.V.%20Dodeka!5e0!3m2!1sen!2snl!4v1638359828973!5m2!1sen!2snl",
+            "logo": "biermijl.jpg",
+            "logo_rond": "",
+            "foto": "biermijl_2023.jpg",
+            "inschrijven": "",
+            "mail": "biermijl@dsavdodeka.nl",
+            "instagram": "https://www.instagram.com/dsavdodeka/?hl=nl",
+            "bepalingen": "",
+            "info_kort": "Elk jaar in oktober vindt de biermijl plaats bij D.S.A.V. Dodeka! De biermijl is het ultieme sportevenement voor studenten, waarbij je 4 rondjes moet lopen en voor elk rondje een biertje moet drinken. Ook kun je er voor kiezen om in een duo mee te doen, waarbij je allebei 2 rondjes loopt en 2 biertjes drinkt.",
+            "info_lang": "Elk jaar in oktober vindt de biermijl plaats bij D.S.A.V. Dodeka. 4 bier, 4 rondjes: durf jij het aan? Mocht je nou niet weten hoe de biermijl werkt, dan staan hieronder alle regels uitgelegd:<br/><br/> 1. Deelname onder de 18 jaar met alcoholvrij bier.<br/> 2. Kom je met de auto, neem iemand mee die nuchter blijft<br/> 3. Deelnemers drinken vier bier en lopen vier rondes. <br/> 4. Bier wordt gedronken voordat wordt begonnen aan de ronde binnen de 10m-zone voor de start.<br/>5. De race begint met het drinken van een bier in de laatste meter van de 10m-zone.<br/>6. Hulpmiddelen en technieken die adten verbeteren zijn verboden.<br/>7. Een bier wordt pas geopend indien de deelnemer de 10m-zone binnenkomt.<br/>8. Indien een deelnemer kotst moet deze een extra ronde lopen. Een extra bier is niet nodig. Indien meer dan een keer wordt gekotst is nog steeds één extra ronde vereist.<br/> 9. Indien een deelnemer kotst, ruimt deze het zelf op.<br/><br/>",
+            "path": "/biermijl"
+        },
+        {
+            "naam": "Dodekathlon",
+            "datum": "29/06/2024",
```

Comment:
Ik zou hier dan ook juni zetten voor de consistentie :) En ik zou deze voor de biermijl zetten zodat ze op de site ook in de goede volgorde komen